client/packages/coldchain/src/Equipment/ListView/CreateAssetModal.tsx

```diff
@@ -70,10 +70,11 @@ const InputRow = ({
 const parseInsertError = (e: unknown) => {
   const message = (e as Error).message;
   if (
-    message.includes('DatabaseError(') &&
-    message.includes('UniqueViolation(') &&
-    message.includes('asset_asset_number_key') &&
-    message.includes('duplicate key')
+    message.includes('DatabaseError(') ||
+    message.includes('UniqueViolation(') ||
+    message.includes('asset_asset_number_key') ||
+    message.includes('duplicate key') ||
+    message.includes('AssetNumberAlreadyExists')
```

Comment:
for me, this is enough:

```suggestion
    message.includes('AssetNumberAlreadyExists')
```

and is easier to understand!

Reply:
yeah I wasn't sure why the others were added XD